docs/CONTRIBUTING.md

```diff
@@ -7,29 +7,134 @@ sure you follow the steps below:
 - Use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
   with `#<your issue number>` as the context for all commits
 - Make sure you are not adding any merge commits to your branch
-- Open a pull request & wait for a review!🩵
+- Open a pull request, include the issue it relates to in the body, for example:
+  `Closes #69`
+- Wait for a review! 🩵
 
-## Developing environment
+For more information on how to get started, read on!
 
-After installing all dependencies with `yarn`, you can run
+## Setting Up Your Build Environment
 
-```sh
-yarn dev
-```
+To build **Vivify** you need the following dependencies:-
 
-to run the development server for Vivify on port 3000 that will (1)
-automatically reload when you make changes to the code, and (2), unlike the
-installed version, not shut down when there are no connected clients.
+**MacOS:**
 
-With the development server running, use
+    TODO: Mac instructions
 
-```sh
-yarn viv
-```
+**Arch Linux/Manjaro:**
 
-instead of your installed Vivify executable. This will (1) connect to the
-development server on port 3000 instead of running the installed server, and (2)
-use the `viv` executable in the repository.
+    sudo pacman -S --needed yarn make zip
+
+**Fedora and derivatives:**
+
+    sudo dnf install yarn make zip
```

Comment:
I'm not sure about this section, isn't it going to blow up a lot if we include every single package manager? Additionally (please let me know if this is too un-inclusive) I would argue that someone who can't figure out how to install `make`, `yarn` and `zip` on their system will have trouble making a valuable contribution to the code anyways.

So I would suggest shortening this to just mention the three dependencies. Same with the part on installing Node below; I would link to `nvm`'s repo/website but not include this level of detail. Also considering the fact that we would have to maintain/update this.

Reply:
Ok I'll revert this to a simple list of deps. I just noticed that it is wrong anyway, because the package on Fedora is called `yarkpkg` not `yarn`

A little anecdote, as I said earlier I took some inspiration from the Ladybird browser project. They have instructions for like 10 different environments as they also don't have any distro packages as it is early development, you have to build it. A couple of popular Linux YouTubers made  videos about it and so they had a flood of non-programmer types coming in to try out this "new browser", which justified the detailed instructions. We probably don't have to worry about that for the foreseeable future. Their issue tracker was fun for a few days there :P

Reply:
Haha, interesting! I think we are fine with this since building is really not required. Either it's packaged for your system, or you download the prebuilt binaries from the latest release :)